mne/io/bti/tests/test_bti.py

```diff
@@ -73,7 +73,8 @@ def test_transforms():
     bti_trans = (0.0, 0.02, 0.11)
     bti_dev_t = Transform('ctf_meg', 'meg', _get_bti_dev_t(0.0, bti_trans))
     for pdf, config, hs, in zip(pdf_fnames, config_fnames, hs_fnames):
-        raw = read_raw_bti(pdf, config, hs, preload=False)
+        with warnings.catch_warnings(record=True):  # weight tables
+            raw = read_raw_bti(pdf, config, hs, preload=False)
```

Comment:
See comment on the catch line, something about 4D weight tables. It's been
that way for a while